docs/4. Product Features/06.  HSM Templates.md

```
@@ -245,65 +211,28 @@ Note: Once the first few videos and follow-ups are working, you can copy the sam
+## Approving multi-lingual HSM templates
 
-<img width="519" alt="image" src="https://github.com/user-attachments/assets/29764c5c-80c9-487c-bbe2-857b86c0486b"/>
 
-Step 2: Create the Template in Another Language (e.g., Hindi)
+1.  Click `Add a new language` on the template list or go into view and "click on add language"
 
-1.  Click `Create Template.`
+<img width="" height="239" alt="Screenshot 2026-07-24 at 1 55 43 PM" src="https://github.com/user-attachments/assets/bb7af925-cc53-404f-a76a-8ebcff027da4" />
 
-2.  Select `Language Variant` from the drop-down.
+3.  Select the langauge that needs to be added
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)